srcpkgs/miniz/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'miniz'
+pkgname=miniz
+version=2.1.0
+revision=1
+build_style=cmake
+short_desc="Single C source file zlib-replacement library"
```

Comment:
Isn't it meant to paste into codebase rather than using as external dependency?

Reply:
After relooking it at it, it appears it may be